edgecompute/examples/traffic-routing/ab-test-caching (EKV)/README.md

```diff
@@ -0,0 +1,20 @@
+# url-redirects (EKV)
+
+*Keyword(s):* ab-test, EKV<br>
+
+This example implements A/B testing at the Edge. It stores test configuration data in EdgeKV and modifies the cache key to allow both a and b variants of the response to be cached at the same URL.
+
```

Comment:
Can we add a note here to clarify that the EdgeWorkers code adds a header in the forward request to the origin, allowing the origin to deliver the `a` or `b` variant as requested?

Additionally, it would be useful to have an example of the data stored in EKV.